packages/components/pie-chip/test/component/pie-chip.spec.ts

```diff
@@ -79,4 +83,171 @@ test.describe('PieChip - Component tests', () => {
+        test.describe('aria-label', () => {
+            test.describe('when an aria close value is provided', () => {
+                test('should render on the component with the correct value', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props: {
+                            ...dismissibleProps,
+                            aria: {
+                                close: 'Chip Close',
+                            },
+                        } as ChipProps,
+                    });
+
+                    const closeButton = page.locator(closeBtnSelector);
+
+                    // Act && Assert
+                    await expect(closeButton).toHaveAttribute('aria-label', 'Chip Close');
+                });
+            });
+
+            test.describe('when an aria close value is not provided', () => {
+                test('should not render on the component', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props: {
+                            ...dismissibleProps,
+                            aria: {
+                                close: '',
+                            },
+                        } as ChipProps,
+                    });
+
+                    const chip = page.locator(componentSelector);
+                    const closeButton = chip.locator(closeBtnSelector);
+
+                    // Act && Assert
+                    await expect(closeButton).not.toHaveAttribute('aria-label', 'Chip Close');
+                });
+            });
+        });
+    });
+
+    test.describe('Chip: aria: attributes', () => {
+        test.describe('aria-live', () => {
+            test('should render on the component with the value `polite`', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieChip, {
+                    props,
+                });
+
+                const chip = page.locator(componentSelector);
+
+                // Act && Assert
+                await expect(chip).toHaveAttribute('aria-live', 'polite');
+            });
+        });
+
+        test.describe('aria-atomic', () => {
+            test('should render on the component with the value `true`', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieChip, {
+                    props,
+                });
+
+                const chip = page.locator(componentSelector);
+
+                // Act && Assert
+                await expect(chip).toHaveAttribute('aria-atomic', 'true');
+            });
+        });
+
+        test.describe('aria-label', () => {
+            test.describe('when passed in', () => {
+                test('should render on the component with the correct value', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props,
```

Comment:
I can see that the value is being passed in inside `props`, but I think this test would be easier to read and understand if a different value was being passed in. Maybe the default values should be removed, and then they are given in the relevant tests?

Reply:
Not sure i follow this one. The values are passed from the prop set up above. I could pass in brand new props here (if that's what you mean?) but the whole test file follows this approach at the moment. I think passing in a brand new set of props does actually make it easier to follow to limit scrolling up and down to check but it does also shorten the file up a bit because your not repeating the same values across each test.

Reply:
Fixed.